extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -37,6 +38,82 @@ class AmpStickyAd extends AMP.BaseElement {
       dev.warn(TAG, `TAG ${TAG} disabled`);
       return;
     }
+
+    // Check `amp-sticky-ad` only have one child with tagName `amp-ad`.
+    this.getRealChildren_ = this.getRealChildren();
```

Comment:
Let's not store the the reference to this array itself however. Instead we should do something like this:
```
const children = this.getRealChildren();
user.assert(children.length == 1 && children[0].tagName == 'AMP-AD', 'amp-sticky-ad must have a single amp-ad child');
/** @const @private {!Element} */
this.ad_ = children[0];
```

Or something like this.